book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental) unless @rentals.include?(rental)
+  end
```

Comment:
- [ ] this method should take person and date as parameters and create a new instance of rental achieving the 'has-many' association
```
 def add_rental(person, date)
    Rental.new(date, self, person)
  end
```